pkgdb2client/__init__.py

```diff
@@ -95,7 +93,7 @@ def ask_password(username=None, bad_password=False):
             username = fedora_cert.read_user_cert()
         except fedora_cert.fedora_cert_error:
             LOG.debug('Could not read Fedora cert, asking for username')
-            username = raw_input("Username: ")
+            username = input("Username: ")
```

Comment:
When testing this on py2 it seems to not behave as I want:
````
>>> t = input('foo:')
foo:test
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "<string>", line 1, in <module>
NameError: name 'test' is not defined
>>> t = raw_input('foo:')
foo:test
````
But indeed on py3 it's the opposite:
````
>>> t = raw_input('foo:')
Traceback (most recent call last):
  File "<input>", line 1, in <module>
NameError: name 'raw_input' is not defined
>>> t = input('foo:')
foo:test
````